README.md

```diff
@@ -88,33 +89,37 @@ A number of code quality practices have been adopted, including:
 
 All V2 efforts are on the [v2](https://github.com/Microsoft/VoTT/tree/v2) branch. The git history for V1 and V2 have significantly diverged; when V2 is complete, master will be updated to reflect V2.
 
-## Where is V1?
+### Where is V1?
 
-Master still reflects The V1 codebase - any fixes or updates can still be made there.  Likewise, all 1.x releases can still be found under [GitHub Releases](https://github.com/Microsoft/VoTT/releases).
+Master still reflects the V1 codebase - any fixes or updates can still be made there.  Likewise, all 1.x releases can still be found under [GitHub Releases](https://github.com/Microsoft/VoTT/releases).
```

Comment:
Same here - we may want to update this now to reference how v1 code will be available in a `v1` branch moving forward.